web/Views/Admin/LandlordList.cshtml

```diff
@@ -1,113 +1,80 @@
-﻿@model LandlordListModel
+﻿@using Microsoft.Extensions.Primitives
+@model LandlordListModel
 
 @{
     ViewData["Title"] = "Landlord List";
 }
 
-<h2>Landlord List</h2>
-
-<a class="btn btn-primary" asp-controller="Landlord" asp-action="RegisterGet" asp-route-createUnassigned="true">Create Unassigned Landlord</a> <br/>
-
-<p></p>
-@using (Html.BeginForm("LandlordList", "Admin", FormMethod.Get))
+@functions
 {
-    <div class="row g-3">
+    string GetCollapseClass()
+    {
+        var isApproved = Context.Request.Query["IsApproved"];

```

Comment:
These properties exist on the view model, is there a reason why we are not just using that?

Reply:
Ah I didn't realise they were there, that simplifies things a lot!